pkg/profile/profile.go

```diff
@@ -2,8 +2,9 @@ package profile
 
 import (
 	"github.com/go-logr/logr"
-	"github.com/weaveworks/profiles/api/v1alpha1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/weaveworks/profiles/api/v1alpha1"
```

Comment:
I tend to name these something like `profilesv1alpha1` rather than leaving the default import `v1alpha1`.